system-tests/README.md

```diff
@@ -1,122 +1,260 @@
-    ```
+_Note for Windows PowerShell, the following commands should be run the the `MVD` root project folder._
 
-- This will start three EDC Connectors, one Registration Service, one HTTP Nginx Server to serve DIDs, Azurite blob storage service and also will seed initial required data using a [postman collection](../deployment/data/MVD.postman_collection.json).
+```powershell
+# Replace path according to your local set up
 
-- `newman` docker container will automatically stop after seeding initial data from postman scripts.
+$Env:REGISTRATION_SERVICE_CLI_JAR_PATH = "c:\RegistrationService\client-cli\build\libs\registration-service-cli.jar"
 
-- EDC Connectors needs to be registered using Registration Service CLI client jar. After publishing RegistrationService locally the client jar should be available under `RegistrationService-Root/client-cli/build/libs` folder.
+# Register Provider
+java -jar $Env:REGISTRATION_SERVICE_CLI_JAR_PATH -s="http://localhost:8184/api" participants add --request="{ \`"name\`": \`"provider\`", \`"supportedProtocols\`": [ \`"ids-multipart\`" ], \`"url\`": \`"http://provider:8282\`" }"
 
-    ```bash
-    export REGISTRATION_SERVICE_CLI_JAR_PATH=registration service client jar path
-    ./system-tests/resources/register-participants.sh
-    ```
+# Register Consumer-EU
+java -jar $Env:REGISTRATION_SERVICE_CLI_JAR_PATH -s="http://localhost:8184/api" participants add --request="{ \`"name\`": \`"consumer-eu\`", \`"supportedProtocols\`": [ \`"ids-multipart\`" ], \`"url\`": \`"http://consumer-eu:8282\`" }"
 
-  for windows powershell
+# Register Consumer-US
+java -jar $Env:REGISTRATION_SERVICE_CLI_JAR_PATH -s="http://localhost:8184/api" participants add --request="{ \`"name\`": \`"consumer-us\`", \`"supportedProtocols\`": [ \`"ids-multipart\`" ], \`"url\`": \`"http://consumer-us:8282\`" }"
+```
 
-    ```powershell
-    $Env:REGISTRATION_SERVICE_CLI_JAR_PATH = "registration service client jar path"
-    # Execute command by copying it from shell script ./system-tests/resources/register-participants.sh or use git-bash to execute this shell script.
-    ```
+Set the environment variable `TEST_ENVIRONMENT` to `local` to enable local blob transfer test and then run `MVD` system test using the following command:
 
-- Run MVD system tests, and for that environment variable `TEST_ENVIRONMENT` must be set to `local` to enable local blob transfer test.
+```bash
+export TEST_ENVIRONMENT=local
+./gradlew :system-tests:test
+```
+
+_Note for Windows PowerShell, the following commands should be used:_
+
+```powershell
+$Env:TEST_ENVIRONMENT = "local"
+./gradlew :system-tests:test
+```
 
-    ```bash
-    export TEST_ENVIRONMENT=local
-    ./gradlew :system-tests:test
-    ```
+> [Storage Explorer](https://azure.microsoft.com/features/storage-explorer/) can be used to connect to the `Azurite` storage container on `127.0.0.1:10000` port and under `consumereuassets`, account transferred blob can be viewed.
```

Comment:
Is `consumereuassets` really the name of the destination account (`transferred blob` seems to imply)? It sounds more like the source account

Reply:
@cpeeyush your thoughts on this?  I did not alter this section from the original

Reply:
yes, it is the name of the storage account for consumer-eu. Here is the list of others https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/blob/e8721ba0e4e7177ebba2dad865cb8536a745f753/system-tests/docker-compose.yml#L133.

The naming is one similar lines(not exact) as we have in the cloud deployment.